api-client/src/main/java/com/desk/java/apiclient/model/OpportunityActivityTypes.java

```diff
@@ -36,9 +36,10 @@
  * Copyright (c) 2015 Desk.com. All rights reserved.
  */
 public class OpportunityActivityTypes {
-    public static final String NOTE = "OpportunityNote";
-    public static final String CALL = "OpportunityCall";
-    public static final String EVENT = "OpportunityEvent";
-    public static final String EMAIL = "OpportunityEmail";
-    public static final String TASK = "OpportunityTask";
+    public static final String NOTE = "note";
```

Comment:
This class should be able to go away now that we have the new enum representation of this.